chain/chunks/src/lib.rs

```diff
@@ -969,6 +966,18 @@ impl ShardsManager {
         }
     }
 
+    fn validate_parts<'a, I>(&mut self, merkle_root: MerkleHash, parts: I) -> Result<(), Error>
+    where
+        I: IntoIterator<Item = &'a PartialEncodedChunkPart>,
+    {
+        let num_total_parts = self.runtime_adapter.num_total_parts();
+        for part_info in parts {
+            // TODO: only validate parts we care about
```

Comment:
Can we create an issue for this if you do not plan to address it in this PR?

Reply:
Created https://github.com/near/nearcore/issues/3662